src/view.js

```diff
@@ -313,6 +313,15 @@ Marionette.View = Backbone.View.extend({
     if (normalizedChildEvents && _.isFunction(normalizedChildEvents[eventName])) {
       normalizedChildEvents[eventName].apply(layoutView, callArgs);
     }
+
+    // call the parent view's proxyEvent handlers
+    var childTriggers = Marionette.getOption(layoutView, 'childTriggers');
+    childTriggers = Marionette._getValue(childTriggers, layoutView);
```

Comment:
By the way @jasonLaster, I think all this stuff should be precomputed rather than done during an event trigger (same for `childEvents`). Thoughts?